airbyte-tests/src/acceptanceTests/resources/postgres_init.sql

```diff
@@ -1,7 +1,7 @@
 CREATE
     TABLE
         id_and_name(
-            id INTEGER,
+            id INTEGER PRIMARY KEY,
```

Comment:
can this be `INTEGER NOT NULL`? (this is a mild preference just to minimize the change)

Reply:
Good call, that should work without requiring any changes to the test logic